Examples/VisionExamples/Constants.swift

```diff
@@ -22,5 +22,13 @@ let listOfExamples: [NamedController] = [
         name: "AR navigation",
         description: "Demonstrates how to setup VisionAR and display default AR route.",
         controllerType: ARNavigationViewController.self
+    ),
+    (
+        name: "POI drawing",
+        description: """
+            Demonstrates how to draw a point of interest on the screen knowing its geogaphical coordinates
```

Comment:
```suggestion
            Demonstrates how to draw a point of interest on the screen knowing its geographical coordinates
```

Reply:
nit:
```suggestion
            Demonstrates how to draw a point of interest on the screen knowing POI's geographical coordinates
```

Reply:
Will fix the typo, thanks. I think it's clear what "its" is referring to, don't you think so?